personal-growth/becoming-a-mentor.md

```diff
@@ -0,0 +1,118 @@
+# Becoming a mentor
+
+Mentoring is an essential aspect of Nebulab: our focus on personal and professional growth is one of
+the main reasons people decide to join us. In turn, this focus allows us to stay relevant and to
+provide world-class consulting services to our clients. As you may imagine, being a mentor in 
+Nebulab is kind of a big deal!
+
+A mentor has many different responsibilities towards the company and their mentees, among them:
+
+- Mentors ensure their mentees are always happy and growing, both as people and professionals.
+- Mentors attend strategic meetings and help define company-wide policies and initiatives.
+- Mentors act as informal project managers for clients, helping them to plan the work to do.
+- Mentors lead by example, embodying the Nebulab values and culture.
+
+With that said, being a mentor is as much of a privilege as it is a chore, and it also has its
+downsides:
+ 
+- Mentors do not get nearly as much time for technical work as individual contributors - 
+  realistically, a mentor can expect to spend half their time writing code and the other half 
+  managing projects and people.
+- Similarly, mentors are rarely able to work on their own Friday projects, because their Fridays are
+  usually eaten by 1:1s and other meetings and company-related initiatives.
+- All this management work is not as visible as technical contributions, so if you're someone who
+  needs a very tight feedback loop (i.e. you need to immediately see the results of your work), you
+  might find mentoring to be frustrating.
```

Comment:
Another downside could be that you will be directly accountable for someone else growth and this could be a big responsibility that not everyone wants to be invested with. What do you think?

Reply:
@kennyadsl added!